UnitySDK/Assets/ML-Agents/Scripts/DemonstrationStore.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.IO;
+using UnityEngine;
+
+
+namespace MLAgents
+{
+    /// <summary>
+    /// Demonstration meta-data.
+    /// </summary>
+    [System.Serializable]
+    public struct DemonstrationMetaData
+    {
+        public int numberExperiences;
+        public int numberEpisodes;
+        public int API;
+    }
+    
+    public class DemonstrationStore
```

Comment:
Two design-focused questions:
- Why can't we support addition of experiences? I imagine a common workflow would be: record some experiences, attempt to train, realized it doesn't do so well and then go back to add more experiences?
- Given that the demo file will be consumed from Python, why not follow the Protobuf definitions as we have for the rest?